include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -80,6 +80,12 @@ struct __subgroup_radix_sort
         {
             return __dpl_sycl::__local_accessor<_KeyT>(__buf_size, __cgh);
         }
+
+        inline static auto
```

Comment:
Let's specify that it is constexpr:

```suggestion
        inline static constexpr auto
```

Meanwhile, inline is optional here (`constexpr` implies it).